src/data/ledger.ts

```diff
@@ -29,14 +48,27 @@ export type LedgerEntry = {
   updated_at: string /*Date*/
 }
 
+export async function getLedgerReports(): Promise<LedgerReport[]> {
+  return await GET('ledger-reports')
+}
+
+export async function getLedgerReportById(id: string): Promise<LedgerReport> {
+  return await GET(`ledger-reports/${id}`)
+}
+
 export async function getPolicyRenewals(): Promise<LedgerReport> {
-  return await GET('ledger?report-type=annual')
+  await processPolicyRenewals()
+  return createLedgerReport(LedgerReportType.annual, new Date().toISOString().split('T')[0])
 }
 
-export async function downloadPolicyRenewals(): Promise<any> {
-  return await GET('ledger?report-type=annual')
+export async function createLedgerReport(reportType: LedgerReportType, date: string): Promise<LedgerReport> {
+  const params: CreateLedgerReportInput = {
+    date: date,
```

Comment:
```suggestion
    date,
```
Maybe you don't like the way this looks, either way is fine.